appinventor/blocklyeditor/src/blocklyeditor.js

```diff
@@ -120,93 +120,125 @@ function unboundVariableHandler(myBlock, yailText) {
         var code = "(let (";
         for (var i in unbound_vars) {
           code += '($' + unbound_vars[i] + ' ' + Blockly.Yail.quotifyForREPL(document.querySelector('input[name="' + unbound_vars[i] + '"]').value) + ') ';
-        };
+        }
         code += ")" + yailText + ")";
         Blockly.ReplMgr.putYail(code, myBlock);
       }
       dialog.hide();
     });
-  };
+  }
 }
 
+/**
+ * Adds an option to the block's context menu to export it to a PNG.
+ * @param {!Blockly.BlockSvg} myBlock The block to export to PNG.
+ * @param {!Array<!Object>} options The option list to add to.
+ */
 Blockly.BlocklyEditor.addPngExportOption = function(myBlock, options) {
   var downloadBlockOption = {
     enabled: true,
-    text: Blockly.BlocklyEditor.makeMenuItemWithHelp(Blockly.Msg.DOWNLOAD_BLOCKS_AS_PNG,
-      '/reference/other/download-pngs.html'),
+    text: Blockly.BlocklyEditor.makeMenuItemWithHelp(
+        Blockly.Msg.DOWNLOAD_BLOCKS_AS_PNG,
+        '/reference/other/download-pngs.html'),
```

Comment:
I just changed this because it went over the [column limit](https://google.github.io/styleguide/jsguide.html#formatting-column-limit). Do you still want me to revert? (I'm totally cool with reverting)

Reply:
Huh. You seem to be right. I don't think I've seen any effort to follow this rule, and I'm personally not wild about it. But it's hard to argue with that.